airflow/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -60,13 +60,17 @@ def execute_query(
         with_event: bool = False,
         wait_for_completion: bool = True,
         poll_interval: int = 10,
+        workgroup_name: str | None = None,
     ) -> str:
         """
         Execute a statement against Amazon Redshift.
 
         :param database: the name of the database
         :param sql: the SQL statement or list of  SQL statement to run
         :param cluster_identifier: unique identifier of a cluster
+        :param workgroup_name: name of the Redshift Serverless workgroup. Mutually exclusive with
```

Comment:
nitpick: i think we should reorder the docstring as well

Reply:
Absolutely :) addressed in 71bea87aff9be72086c85d99862568701e014bc4